assets/data-port/import/data/actions.js

```diff
@@ -0,0 +1,281 @@
+import {
+	API_BASE_PATH,
+	FETCH_FROM_API,
+	START_FETCH_IMPORT_DATA,
+	SUCCESS_FETCH_IMPORT_DATA,
+	ERROR_FETCH_IMPORT_DATA,
+	START_START_IMPORT,
+	SUCCESS_START_IMPORT,
+	ERROR_START_IMPORT,
+	START_UPLOAD_IMPORT_DATA_FILE,
+	SUCCESS_UPLOAD_IMPORT_DATA_FILE,
+	ERROR_UPLOAD_IMPORT_DATA_FILE,
+} from './constants';
+
+import { normalizeImportData } from './normalizer';
+
+/**
+ * @typedef  {Object} FetchFromAPIAction
+ * @property {string} type               Action type.
+ * @property {Object} request            Object that is used to fetch.
+ */
+/**
+ * Fetch action creator.
+ *
+ * @param {Object} request Object that is used to fetch.
+ *
+ * @return {FetchFromAPIAction} Fetch action.
+ */
+export const fetchFromAPI = ( request ) => ( {
```

Comment:
Some of these common fetch actions and controls could be moved to files shared between the setup wizard and importer, exporter stores to have less duplication. Should be safe to make the action names generic too, ie `SUCCESS_FETCH_IMPORT_DATA` => `FETCH_SUCCESS`.

Reply:
I agree. I'll do it in a followup PR if that's okay just because this one is getting a bit big already.